src/Features/CSharp/Portable/Completion/KeywordRecommenders/NullKeywordRecommender.cs

```diff
@@ -14,37 +12,9 @@ public NullKeywordRecommender()
         {
         }
 
-        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
-        {
-            var syntaxTree = context.SyntaxTree;
-
-            return
-                context.IsAnyExpressionContext ||
-                context.IsStatementContext ||
-                context.IsGlobalStatementContext ||
-                IsInSelectCaseContext(context, cancellationToken);
-        }
-
-        private bool IsInSelectCaseContext(CSharpSyntaxContext context, CancellationToken cancellationToken)
```

Comment:
this is deleted following discussion here: https://github.com/dotnet/roslyn/issues/25293#issuecomment-371074889